extras/DevelopersREADME.md

```diff
@@ -0,0 +1,62 @@
+DQD Developers README
+====================
+
+Dev Setup 
+====================
+1. R setup: https://ohdsi.github.io/Hades/rSetup.html  
+2. Local OMOP CDM setup 
+
+   If you already have a CDM available for development work/testing, you may skip this step 
+   
+   a. Install Postgres and create a localhost server 
+   
+   b. Create a new database in localhost for your test CDM, and create a schema in that database for the CDM tables 
+   
+   c.Using the CDMConnector package:
+
+       i.  Download a sample OMOP CDM into a DuckDB database, as documented here 
```

Comment:
this formatting got messed up; it's rendering as code instead of sub-bullets. also make sure to add in the hyperlinks out to CDMConnector docs on each line.